### ISSUE TITLE
Analytics New KPI dashboard request & KPI dashboard update request for [Burial Form 21P-530EZ]

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#80861]
# Request a New Standardized KPI Dashboard or an Update to an Existing Dashboard
## What this form is for

Use this ticket template to request a new standardized KPI dashboard or an update to an existing KPI dashboard on Domo for your product. 

**Tell us what you need done:**
- [x] [New Dashboards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/analytics-request-kpi-dashboard.md#for-new-dashboards)
- [ ] [Updates to Existing Dashboards](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/.github/ISSUE_TEMPLATE/analytics-request-kpi-dashboard.md#for-updates-to-existing-dashboards)

Standardized dashboards are currently available for [Forms](https://va-gov.domo.com/page/447193050?userId=853813555), [Content](https://va-gov.domo.com/page/426422632), & [Search](https://va-gov.domo.com/page/1964748112) products only. To see if you should request a standardized dashboard, check [Should I request a standardized KPI dashboard](https://docs.google.com/presentation/d/16mRcslbkWtW44rlsfQGXZnHtQ3GoLjn6GpFnd3Gvva8/edit#slide=id.p). If you’re not sure which dashboard(s) your product(s) qualify for, check the [Eligibility Dashboard](https://va-gov.domo.com/page/1550814739).

If your VA.gov product is not eligible for a standardized dashboard, you may be able to request an “edge case” dashboard. Please reach out to the Platform Analytics & Insights team on #vfs-platform-support for more information on this process.

### Expected delivery timelines

The Analytics team will notify you once this ticket has been pulled into a sprint. 

## For New Dashboards

In order to pull the correct data for your product, we need your help to define what pages and events to include, in addition to basic product info. We’ll need a filled out [KPI request template](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988) before we can pull this ticket into our sprint.

- [x] Make a copy of the [Google Sheet](https://docs.google.com/spreadsheets/d/1gipo-h3GCeVPsw6t6tEHJM5dtJMvotVm0LUgbpAeqZY/edit#gid=981224988)
- [x] Complete worksheet as instructed
- [x] Link your completed spreadsheet here: https://docs.google.com/spreadsheets/d/1ptfQI71LP85un1kXfrwOwzwbl9VYKrjBVHxLCBlyAgQ/edit?usp=sharing
- [x] Check your sharing settings to make sure the Analytics team will be able to access it
- [x] **Add your team label to follow progress**

**Additional Info For Both Options**

- Please use the [KPI Framework](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/Analytics%20Playbook/va-gov-platform-analytics-kpi-framework.pdf) for more information on the metrics used in the Standard KPI dashboards. 
- If you need any analytics support, read through the [How to request Analytics support](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Analytics-customer-support-guidelines.1586823275.html#Analyticscustomersupportguidelines-4.RequestaKPIDashboardinDomo).
- If you do not have Domo access, please [request access here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+access-request&template=analytics-request-google-analytics-domo-access.md&title=Request+access+to+Google+Analytics+and%2For+Domo).

**Comment by johnny-jesensky-adhoc** [comment]
Hi @steele-lm  - The form has been added to the Forms KPI Dashboard, though it won't start appearing in Domo next Monday, May 13th. Our data refreshes weekly, and as the form was added today, we'll have to wait til the next refresh happens over the weekend.

We created a separate ticket to check back in on the confirmation. https://github.com/department-of-veterans-affairs/va.gov-team/issues/82436

FYI @Blivaditis , @ChristineYoungZA , @dorseypa1

**Comment by steele-lm** [comment]
@johnny-jesensky-adhoc Thanks so much! Excited to see the dashboard in Domo.

**Comment by steele-lm** [comment]
@johnny-jesensky-adhoc I'm not seeing the Domo dashboard for the burial 21-530EZ form yet. Do you know if the data has refreshed yet?

**Comment by johnny-jesensky-adhoc** [comment]
Hi @steele-lm -- we had a formatting error when loading the forms that we didn't catch until earlier this week - we've re-uploaded the data, but things won't display until the next data refresh cycle, which is next Sunday. You can track the latest on this in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82436), which we have in our current sprint.

We have a note to check back in and confirm things are displaying in Domo next Monday, May 20th -- apologize for the delay!
